src/components/lyne-toast/styles/lyne-toast.component.scss

```diff
@@ -0,0 +1,113 @@
+// Default component properties, defined for :host. Properties which can not
+// travel the shadow boundary are defined through this mixin, available for shared
+// and default mode
+@include host-component-properties;
+
+:host {
+  --color-default-text: var(--color-white-default);
+  --toast-background-color: var(--color-midnight-default);
+  --toast-border-radius: #{pxToRem(var(--border-radius-4x))};
```

Comment:
I suggest to move pxToRem to the usages (as in other components). e.g. if you make a further calculation, it won't create a lot of outputs.